tools/gpm

```diff
@@ -74,13 +75,26 @@ set_dependencies() {
   done < <(echo "$deps")
   wait
 
-  while read package version; do
-    echo ">> Building package "$package""
-    go install "$package"
-  done < <(echo "$deps")
+  if [[ $no_install != true ]]
```

Comment:
Sure, I was just following the style guide they set forth since this is not our script.